Whiskey/Functions/Invoke-WhiskeyTask.ps1

```diff
@@ -259,7 +263,24 @@ function Invoke-WhiskeyTask
                 $parameter.Remove('Debug')
             }
 
-            & $task.CommandName @parameter
+            $outVariable = $commonProperties['OutVariable']
+
+            if ($outVariable)
+            {
+                $taskOutput = & $task.CommandName @parameter

```

Comment:
Maybe `Tee-Object` so people can at least see the output, too?

```
& $task.CommandName @parameter | Tee-Object -Variable 'taskOutput'
```

The only problem with this is `Tee-Object` fails if `$WhatIf` is `True`.

Reply:
> The only problem with this is Tee-Object fails if $WhatIf is True.
Is that a problem here? We don't support running a Whiskey build with `-WhatIf` nor does `Invoke-WhiskeyTask` support `ShouldProcess`.

However, if we did end up using `Tee-Object` then we'd also have to add another common property, `sensitive`. One use I'd have for `OutVariable` is to run a command that gets a secret to pass to another task. I'd need to use `sensitive: true` to suppress that task's output and not leak the secret in the log.

Reply:
I tried implementing this with `Tee-Object` and a common `sensitive` property but I didn't like it as it made actually calling the task really hairy with having to support all combinations of those properties. Given that, I opted to forgo that addition as I don't think it's something we need right now.